src/Orleans.Core/Serialization/SerializationManager.cs

```diff
@@ -558,7 +577,7 @@ internal static void GetSerializationMethods(Type type, out MethodInfo copier, o
             copier = null;
             serializer = null;
             deserializer = null;
-            foreach (var method in type.GetMethods(BindingFlags.Static | BindingFlags.Public | BindingFlags.NonPublic))
+            foreach (var method in type.GetMethods(BindingFlags.Instance | BindingFlags.Static | BindingFlags.Public | BindingFlags.NonPublic))
```

Comment:
Probably a case of "who cares", but will this fail at runtime if we have a mix of static & instance methods for copy/serialize/deserialize on a given type?